{{ cookiecutter.package_name }}/.coveragerc

```diff
@@ -0,0 +1,34 @@
+[tool.coverage.run]
+omit = [
+ "{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init*",
+ "{{ cookiecutter.module_name }}/conftest.py",
+ "{{ cookiecutter.module_name }}/*setup_package*",
+ "{{ cookiecutter.module_name }}/tests/*",
+ "{{ cookiecutter.module_name }}/*/tests/*",
```

Comment:
Test coverage should be measured.

Is this how it is in the OA template?

Reply:
apparently yes: https://github.com/OpenAstronomy/packaging-guide/blob/main/%7B%7B%20cookiecutter.package_name%20%7D%7D/pyproject.toml#L79

Reply:
Also astropy ignore them: https://github.com/astropy/astropy/blob/main/pyproject.toml#L267

Reply:
Astropy shouldnt, no one should ignore them. Coverage of tests mean you check you are running your tests

Reply:
Isn't coverage testing what the tests cover? So checking that they're covered by themselves doesn't make loads of sense?

Reply:
Only if you run the tests correctly and have written correct tests.

Reply:
I think that we probably should, I haven't noticed anything problematic about it on sunpy and when we have lots of builds where we only run some of the tests on some jobs / platforms / OSes etc. The combined coverage over all our jobs is a good check to make sure we are actually running all the tests which get conditionally run.

Reply:
That isn't true. David has found several tests that were not run or asserted in the tests over the years. 

Coverage of test files ensures that.

Reply:
I have always ignored test files in coverage but that actually makes a lot of sense...

Reply:
I finally managed to dig up the blog I remember about this: https://nedbatchelder.com/blog/201908/dont_omit_tests_from_coverage.html